Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml

```diff
@@ -127,27 +127,71 @@
   - type: StaticPrice
     price: 3500
 
+# Start DeltaV removals
```

Comment:
This wasn't removed though, you just changed some of the fields
Add comments on what was changed on the relevant lines instead of adding a whole comment block